### PR TITLE
Updated Ubiquiti Identity Desktop to 1.1.3.91

### PR DESCRIPTION
**Pull request by bearmannl**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/182580)

### CODE REVIEW

File: manifests/u/UbiquitiInc/IdentityDesktop/1.1.3.91/UbiquitiInc.IdentityDesktop.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+CopyrightUrl: https://www.ui.com/legal/
+ShortDescription: Identity Desktop is Ubiquiti's free tier Identity application for Windows.
+Description: Unifi Identity Desktop is Ubiquiti's new free tier Identity application for Windows. Allowing for One-Click WiFi, One-Click VPN, and more.
+Moniker: unifi-identity-desktop
+Tags:
+- unifi
+- identity
+- wifi
+- vpn
+ReleaseNotes: UniFi Identity Endpoint Windows 1.1.2.74 includes the following improvements.
```
Comment (Trenly) [inline manifests/u/UbiquitiInc/IdentityDesktop/1.1.3.91/UbiquitiInc.IdentityDesktop.locale.en-US.yaml:24]:
```suggestion
ReleaseNotes: >-
  UniFi Identity Endpoint Windows 1.1.2.74 includes the following improvements.
  * Ensure that the One-Click WiFi and VPN connection remains active when switching between consoles.
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Hi @bearmannl,

I'm having trouble finding this package on the developer's [download page](https://www.ui.com/download). Is there a better page?

**Comment by bearmannl** [comment]
> Hi @bearmannl,
> 
> I'm having trouble finding this package on the developer's [download page](https://www.ui.com/download). Is there a better page? 

Hi @stephengillie,

If you go to the release notes URL, there is a download link below the release notes.

That link translates directly into the download link after you click it.

**Comment by stephengillie** [comment]
Thank you for helping me to find it. I wish they would put the link in a more obvious location.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**